src/index.html

```diff
@@ -1,12 +1,412 @@
+          alt="imacIcon"
+          class="card__icon"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__product">
+          Product code: 195434
+        </p>
+
+
+        <div  class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__word">Price:</p>
+          <p class="card__digit">$2,199</p>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="card__item"
+            href="#buy"
+          >
+            Buy
+
+          </a>
+        </div>
+        </div>
+      </section>
+
+      <section class="card card--3">
+
+        <img
+          src="images/imac.jpeg"
+          alt="imacIcon"
+          class="card__icon"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__product">
+          Product code: 195434
+        </p>
+
+
+        <div  class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__word">Price:</p>
+          <p class="card__digit">$2,199</p>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="card__item"
+            href="#buy"
+          >
+            Buy
+
+          </a>
+        </div>
+        </div>
+      </section>
+
+      <section class="card card--4">
+
+        <img
+          src="images/imac.jpeg"
+          alt="imacIcon"
+          class="card__icon"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__product">
+          Product code: 195434
+        </p>
+
+
+        <div  class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__word">Price:</p>
+          <p class="card__digit">$2,199</p>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="card__item"
+            href="#buy"
+          >
+            Buy
+
+          </a>
+        </div>
```

Comment:
I think you can use just <a> link withou div that wraps it, I used only <a>,  it will give you much better view of code.

Reply:
<div's were remowed and classes was replaced